web/src/components/organisms/Settings/MyIntegrationDetails/hooks.ts

```diff
@@ -0,0 +1,134 @@
+import { useCallback, useMemo } from "react";
+
+import { WebhookTrigger } from "@reearth-cms/components/molecules/MyIntegrations/types";
+import integrationHook from "@reearth-cms/components/organisms/Settings/MyIntegrations/hooks";
+import {
+  useCreateWebhookMutation,
+  useUpdateIntegrationMutation,
+  useUpdateWebhookMutation,
+  useDeleteWebhookMutation,
+} from "@reearth-cms/gql/graphql-client-api";
+
+type Params = {
+  integrationId?: string;
+  webhookId?: string;
+};
+
+export default ({ integrationId, webhookId }: Params) => {
+  const { integrations } = integrationHook();
+
+  const selectedIntegration = useMemo(() => {
+    return integrations.find(integration => integration.id === integrationId);
+  }, [integrations, integrationId]);
+
+  const webhookInitialValues = useMemo(() => {
+    if (!selectedIntegration || !selectedIntegration.config.webhooks || !webhookId) return;
+    const selectedWebhook = selectedIntegration.config.webhooks.find(
+      webhook => webhook.id === webhookId,
+    );
+    if (!selectedWebhook) return {};
```

Comment:
Choosing one fallback return type I think is better and then you can handle just two possible outcomes (undefined/initialValues OR emptyObject/initialValuesObject)

Reply:
Empty object, as it is more standard for antd form initial value.